inductiva/resources/resources.py

```diff
@@ -0,0 +1,62 @@
+"""Functions to manage or retrieve user resources."""
+from absl import logging
+import inductiva
+from inductiva.client.apis.tags import instance_api
+from inductiva import resources
+
+
+def get_storage_size():
+    """Returns the total storage size in GB."""
+    try:
+        api = instance_api.InstanceApi(inductiva.api.get_client())
+        instance_price = api.get_storage_size()
+        return instance_price.body
+    except inductiva.client.ApiException as api_exception:
+        raise api_exception
+
+
+def list():
```

Comment:
`list` is a name too short and general, why not `list_machine_groups`?

Reply:
this is how it was decided in a previous meeting. the goal is to call it with `inductiva.resources.list()`.

Reply:
But also with the name list, pylint raises an error, so maybe it's better to rename it to `list_machine_groups`?